kernel/content/ezcontentoperationcollection.php

```diff
@@ -1017,6 +1017,12 @@ static public function changeHideStatus( $nodeID )
                 eZContentObjectTreeNode::hideSubTree( $curNode );
         }
 
+        // notify cache system about visibility change
+        ezpEvent::getInstance()->notify('content/cache', array(
```

Comment:
maybe move this inside ` if ( is_object( $curNode ) )`? otherwise nothing is actually changed